core_backend/tests/rails/test_safety.py

```diff
@@ -17,24 +18,59 @@ def read_test_data(file: str) -> list[str]:
         return f.read().splitlines()
 
 
+@pytest.fixture
+def response() -> UserQueryResponse:
+    """Returns a dummy response"""
+    return UserQueryResponse(
+        query_id=1,
+        content_response=None,
+        llm_response="Dummy response",
+        feedback_secret_key="feedback-string",
+    )
+
+
 @pytest.mark.parametrize("prompt_injection", read_test_data(PROMPT_INJECTION_FILE))
-def test_prompt_injection_found(prompt_injection: pytest.FixtureRequest) -> None:
+def test_prompt_injection_found(
+    prompt_injection: pytest.FixtureRequest, response: pytest.FixtureRequest
+) -> None:
     """Tests that prompt injection is found"""
-    assert classify_safety(prompt_injection) == SafetyClassification.PROMPT_INJECTION
+    question = UserQueryRefined(
+        query_text=prompt_injection, query_text_original=prompt_injection
+    )
+    _, response = _classify_safety(question, response)
+    assert response.state == ResultState.ERROR
+    assert (
+        response.debug_info["safety_classification"]
+        == SafetyClassification.PROMPT_INJECTION.value
+    )
```

Comment:
llm classified it as safe :/ 

<img width="864" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/b0a5b1a3-5165-4478-96b4-520b2c8a45a1">

Reply:
I am hoping not to invest too much in improving safety rail for a bit. We may not need it if the other rails works well.